### PR TITLE
Fix #1185

### LINKED ISSUE #1185: Should {path} in access logs show value for {>Caddy-Rewrite-Original-URI}
This is an related to  #1153 assuming Pull Request #1184 is merged.

In an access log the placeholder  `{>Caddy-Rewrite-Original-URI}` can be used with a fastcgi site that does a rewrite to output the origional path requested.  {path} will output the path after redirection

eg.

with a caddyfile

```
localhost:1234 {
log /  access.log "{path} {>Caddy-Rewrite-Original-URI} "

rewrite / {
 r (.*)
 to /redir.php
}

 fastcgi / localhost:34345  php
 root  D:\webroot\phpcgi
}
```

for the url  http://localhost:1234/customers/department/sales/2233 the log will show

> /redir.php /customers/department/sales/2233 

So, there is an argument here for {path} in the caddy file to log the same value as {>Caddy-Rewrite-Original-URI} as really it is the requested path that is of interest in a log rather than the rewritten path.

Is it worth considering another log placeholder such as `{rewrite_path}` which would contain what `{path}` currently contains, ie. the path after rewriting, and have `{path}` output what `{>Caddy-Rewrite-Original-URI}` does?

### PR DESCRIPTION
**Pull request by javimuu**

### CODE REVIEW

**Comment by elcore** [comment]
Hey,

you can't do that! I am not working anymore on this issue --  It's a proof of concept!!!

**Comment by mholt** [comment]
Wait, ... well, this PR was fine, but it needs some work -- what's a proof of concept, @elcore?

Issue #1185 is still in the discussion phase. We need to figure out how other web servers handle this, I'm not sure what "most" people would expect here.

**Comment by elcore** [comment]
@mholt I closed this PR because it was created by a spammer (just look at his history)

> what's a proof of concept

The code! Replacing {path} with {rewrite_path}

File: caddyhttp/httpserver/replacer.go [hunk 1]
```diff
@@ -221,8 +221,12 @@ func (r *replacer) getSubstitution(key string) string {
 		}
 		return host
 	case "{path}":
-		return r.request.URL.Path
+		return r.request.Header.Get("Caddy-Rewrite-Original-URI")
```
Comment (elcore) [inline caddyhttp/httpserver/replacer.go]: I know. That's the reason I never created a PR -- It's a proof of concept!

**Comment by mholt** [comment]
Elcore, it's fine. No harm done. I'm glad people are at least looking at contributing. I agree it would be helpful if we could finish the discussion in the issue first. @javimuu feel free to join our discussion in #1185 until we can reach consensus about how to handle this. :)

**Pull request closed without merging**